LibertyDSNP/example-client
# start of logging info

Purpose
---------------
Feature

[https://www.pivotaltracker.com/story/show/179200375](https://www.pivotaltracker.com/story/show/179200375)

In order to help developers follow what the example client is doing, we want copious logging to the console showing what the app is doing.

Solution
---------------
- Logging Added to the following areas:
- Login (Wallet connection, registration search, handle selection, handle creation)
- Logout
- New batch discovered
- activity content saved, announcement signed/created, batch created/saved, batch publication notification sent to chain
  - For new post, reply, graph changes, profile updates, etc...
- Show transaction confirmation with block number and hash

Should be separated out into an example logging module.

Something like (find better naming): `exlog.info("msg", dataObj)`

What to show in the logs? Reasonable string message + data.

Change summary
---------------
* Created logInfo.ts file that contains cases and console logs.
* call those console logs in the appropriate places in the code.


Steps to Verify
----------------
1. Check that the logs are in the correct places and that they are explaining what we want to.
2. Check that we are logging the correct things (do we want to log anything else or not log some of the things we are logging?).
3. Should we be logging "Reading batch announcement: ..." That is so many console logs
